src/azure-cli/azure/cli/command_modules/vm/_help.py

```diff
@@ -129,11 +129,20 @@
 examples:
   - name: Create a disk encryption set
     text: az disk-encryption-set create -g MyResourceGroup -n MyDiskEncryptionSet --key-url MyKey --source-vault MyVault
+  - name: Create a disk encryption set. (autogenerated)
+    text: |
+        az disk-encryption-set create --key-url MyKey --location westus2 --name MyDiskEncryptionSet --resource-group MyResourceGroup --source-vault MyVault
+    crafted: true
```

Comment:
Duplicate

Reply:
removed the auto-generated one and updated the huamn written.